recipes/cvxpy/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install 
```

Comment:
This and the `bld.bat` above are not required as you are using the script tag below.